.github/workflows/build_wheels.yml

```diff
@@ -0,0 +1,232 @@
+name: Build Wheels
+
+on:
+  workflow_call:
+    inputs:
+      auto_bump:
+        description: 'Bump version automatically'
+        type: boolean
+        required: false
+        default: false
+
+jobs:
+  linux:
+    name: Build ${{ matrix.platform.runner}} ${{ matrix.platform.target }}
+    runs-on: ${{ matrix.platform.runner }}
+    strategy:
+      matrix:
+        platform:
+          # older ubuntu to avoid messing with glibc version
+          - runner: ubuntu-22.04
+            target: x86_64
+            manylinux: auto
+            interpreter: "3.9 3.10 3.11 3.12 3.13"
+          - runner: ubuntu-22.04
+            target: aarch64
+            manylinux: manylinux_2_28
+            interpreter: "3.9 3.10 3.11 3.12 3.13"
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.13"
+
+      - run: pip install -U twine
+
+      - name: Install required packages
+        run: |
+          sudo apt update
+          sudo apt install pkg-config gcc-aarch64-linux-gnu g++-aarch64-linux-gnu -qy
+
+      - uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: stable
+          target: ${{ matrix.platform.target}}-unknown-linux-gnu
+      - uses: Swatinem/rust-cache@v2
+
+      - name: Bump version in Cargo.toml
+        if: ${{ inputs.auto_bump }}
+        uses: ./.github/actions/bump_version
+
+      - name: Build wheels
+        uses: PyO3/maturin-action@v1
+        with:
+          target: ${{ matrix.platform.target }}
+          args: --release --out dist --interpreter ${{ matrix.platform.interpreter }}
+          sccache: ${{ !startsWith(github.ref, 'refs/tags/') }}
+          manylinux: ${{ matrix.platform.manylinux }}
+
+      - name: Validate Python package distributions
+        run: twine check --strict dist/*
+
+      - name: Install built wheel
+        if: matrix.platform.target == 'x86_64'
+        run: |
+          pip install outlines_core --no-index --find-links dist --force-reinstall
+          python -c "import outlines_core"
+
+      - uses: actions/upload-artifact@v4
+        with:
+          path: dist/*.whl
+          name: wheels-linux-${{ matrix.platform.target }}
+
+  windows:
+    name: Build ${{ matrix.platform.runner}} ${{ matrix.platform.target }}
+    runs-on: ${{ matrix.platform.runner }}
+    strategy:
+      matrix:
+        platform:
+          - runner: windows-latest
+            target: x86
+            alias-target: i686-pc-windows-msvc
```

Comment:
Are Windows 32 bits programs still a thing?

Reply:
We're simply supporting fully what's being supported before. If we'll decide to drop it at some point, it can be a separate decision.